lib/MusicBrainz/Server/Controller/Genre.pm

```diff
@@ -0,0 +1,42 @@
+package MusicBrainz::Server::Controller::Genre;
+use Moose;
+
+use MusicBrainz::Server::Constants qw( %ENTITIES );
+
+BEGIN { extends 'MusicBrainz::Server::Controller'; }
+
+sub list : Path('/genres') Args(0) {
+    my ($self, $c) = @_;
+
+    my @genres = $ENTITIES{tag}{genres};
```

Comment:
Should be `$genres`, and `genres => $genres` in `component_props` below, since it's an array ref. What you have works accidentally, since you're creating a list with one item (the array ref), but it's misleading and can cause bugs later.